src/content/desci/index.md

```diff
@@ -8,77 +8,268 @@ sidebar: true
+
+This is due in part to the gatekeeping of knowledge and resources by third party arbiters, as well as the inability of the general public to participate in and understand the scientific process. 
+
+Decentralized science can help to address these issues by allowing for more diverse funding sources (from DAOs, quadratic donations to crowdfunding and more), easier access to data and methods, and by providing incentives for reproducibility.
+
+​​
+
+
+## Why we need science
+
+Science is a process that allows us to create and translate knowledge in order to improve the quality of life for humanity as a whole. It also provides solutions to some of the most complex problems we face. In order to be trustworthy, knowledge produced by science must be replicable and open to scrutiny. This process should be impartial and available to the public. Any knowledge that cannot be replicated or tested for falsifiability should not be considered as the truth.
+
+By creating knowledge that is impartial and available to the public, we enable ourselves to make fundamental technological advancements. These advancements give us the power to more effectively navigate and manipulate our environment, leading to vast improvements in quality of life. As we continue to create impartial knowledge and make technological advancements, we find ourselves capable of managing increasingly complex systems. Today, we are confronted with species-level existential risks, such as climate change and global pandemics. Science is the only process we have that offers equitable positive outcomes.
+
+
+## How DeSci improves science
+
+**List of key problems in science and how decentralized science can help to address these issues**
+
+
+
+* bureaucratic funding structures -> allow for more diverse funding sources (quadratic donations, DAOs, crowdfunding etc.) 
+* replication crisis -> allowing for easier access to data and methods, and by providing incentives for reproducibility.
+* misaligned incentives -> aligning incentives more closely with the goals of science. 
+* poor data quality -> incentivising transparent data sharing and quality control. 
+* publication bias -> enabling publication in a wider range of outlets. 
+* p-hacking -> incentivising transparent and honest reporting of results.
+* cherry picking -> requiring publication of all results, including negative or inconclusive ones. 
+* HARKing: hypothesizing after the results are known -> providing incentives for transparent and honest reporting of results.
+
+A comparison to traditional science
+
+
+<table>
+  <tr>
+   <td><strong>DeSci</strong>
+   </td>
+   <td><strong>Traditional science</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>You raise funding from a global community.
+   </td>
+   <td>You raise funding from a national agency.
```

Comment:
sometimes - my experience has been to attract funding more regularly from societies, private businesses, charities and international funding agencies than national funding agencies. Reducing the current landscape down to national funding agencies seems a bit narrow.

Reply:
good point, would from a centralized agency vs decentralized community be a better comparison?

Reply:
IMO the value that DeSci adds to the science funding model comes from the fact that the public can influence the distribution of funds via e.g. QF or at least delegate their influence to a trusted party, rather than the decisions being concentrated entirely in very small, centralized committees --> better alignment between community values and capital allocation.

Also that Gitcoin, Uniswap etc have shown that institutions are willing to fund "public goods" generally and that this could be a new source of science funding if the downstream incentive model is organized correctly. 

Something like the following might work:

| ...| ...|
|--------------------------------------------------------------------|--------------------------------------------------------------------------|
|Distribution of funds is controlled by small, closed, centralized groups | Distribution of funds is determined by the public using mechanisms such as quadratic voting|

Reply:
I agree with the table change but would change the language in the DeSci section from "is" to "can be".

A decentralized system can still have participants manifested as small closed centralized groups that control funds